tools/contributed/traas/src/main/java/de/tudresden/sumo/util/SumoCommand.java

```diff
@@ -130,7 +130,7 @@ public SumoCommand(Object input1, Object input2, Object input3, Object[] array,
 
                 cmd.content().writeUnsignedByte(Constants.REQUEST_DRIVINGDIST);
 
-            } else if ((Integer) input1 == Constants.CMD_GET_SIM_VARIABLE && (Integer) input2 == Constants.POSITION_CONVERSION) {
+            } else if (input1 == Constants.CMD_GET_SIM_VARIABLE && input2 == Constants.POSITION_CONVERSION) {
 
                 if ((Integer)output_type == Constants.POSITION_ROADMAP) {
```

Comment:
You missed one, can you please add this one too?

Reply:
Thanks a lot, it is still much duplicated code but at least the casts are gone.

Reply:
@behrisch Thanks for pointing out this oversight.  I probably missed it because it didn't have a space between the cast type and the expression.
You can take another look now.